chromium/pages/cancel/index.html

```diff
@@ -8,11 +8,18 @@
     <link rel="stylesheet" href="style.css" type="text/css" />
   </head>
   <body>
-    <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
-    <p data-i18n="cancel_he_blocking_explainer"></p>
-    <p id="url-paragraph"><span id="url-label">URL: </span><a href="#" id="originURL"></a></p>
-    <script src="../translation.js"></script>
-    <script src="../util.js"></script>
-    <script src="ux.js"></script>
+    <div class="content">
+      <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
+      <p data-i18n="cancel_he_blocking_explainer"></p>
+      <p id="url-paragraph">
+        <span id="url-label">URL: </span><a href="#" id="url-value">PLACEHOLDER</a>
+      </p>
+      <form method="get" action="#" id="url-form">
+        <button name="open" formaction="#" id="open-url-button">Open page insecurely</button>
```

Comment:
It would be better if it was more obvious that this option permanently whitelists the website.

Reply:
Should the notice be part of the button or in subtext underneath?

Reply:
Make the button say: "Open insecurely and do not ask again".

Reply:
The button text has been updated.